### PR TITLE
Fix issues brought up with Schedule in the PED

### PR DESCRIPTION
**Pull request by iaso1774**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ Examples of usage:
 ### Add Deadline to a Manga: `schedule`
 Adds a deadline to an existing manga. `-b` stands for by-date (in case you are wondering why it is not `-d`, which is used for deleting in other commands). 
 
+It should be noted that the deadline does not need to follow any set format.
+This allows for greater flexibility in setting deadlines (e.g. `20 Jan 2024`, `Tuesday`, `After Mum's bday`, `in 2 days`).
+It is up to the user to define what is valid.
+
```
Comment (sarahchow03) [inline docs/UserGuide.md:86]: I suggest that you place this in a note under the command. See the other notes in the User Guide on how to do this.

It should look like this in Markdown:
{A8C1F090-1B72-44D6-8382-483644BDEAE7}

**Review by sarahchow03 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (iaso1774) [reply docs/UserGuide.md:86]: Ok, I will do this and commit the changes shortly.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ Examples of usage:
 `sales -a Izumi Tsubaki -m Gekkan Shoujo Nozaki-kun -q 1700 -p 12.90`
 
 ### Add Deadline to a Manga: `schedule`
-Adds a deadline to an existing manga. `-b` stands for by-date (in case you are wondering why it is not `-d`, which is used for deleting in other commands). 
+Adds a deadline to an existing manga. `-b` stands for by-date (in case you are wondering why it is not `-d`, which is used for deleting in other commands).
```
Comment (sarahchow03) [inline docs/UserGuide.md:81]: Similar to a comment I got for the 'sales' data, I think you should also specify that the deadline is overwritten if ran again?

**Review by sarahchow03 — CHANGES_REQUESTED** [review]
Apologies, one more comment just so that we can cover all bases with this command!

File: docs/UserGuide.md (hunk 2 above)
Comment (iaso1774) [reply docs/UserGuide.md:81]: Good point, I'll make that change.

**Review by sarahchow03 — APPROVED** [review]
LGTM!

**Pull request merged**